downstream/titles/hub/configuring-private-hub-rh-certified/docinfo.xml

```diff
@@ -1,7 +1,7 @@
-<title>Managing Red Hat Certified and Ansible Galaxy collections in automation hub</title>
+<title>Managing Red Hat Certified, validated and Ansible Galaxy content in automation hub</title>
```

Comment:
Flag name change to DPM: Pantheon and customer portal will need updates.
Should it have the Oxford comma? 
Managing Red Hat Certified, validated, and Ansible Galaxy content in automation hub

Reply:
I have nothing against Oxford commas. It makes more sense with them I think.